src/V3Inline.cpp

```diff
@@ -476,175 +439,225 @@ class InlineRelinkVisitor final : public VNVisitor {
 };
 
 //######################################################################
-// Inline state, as a visitor of each AstNode
+// Module inliner
+
+namespace ModuleInliner {
+
+// Connect the given port 'nodep' (being inlined into 'modp') to the given
+// expression (from the Cell Pin)
+void connectPort(AstNodeModule* modp, AstVar* nodep, AstNodeExpr* pinExprp) {
+    UINFO(6, "Connecting " << pinExprp);
+    UINFO(6, "        to " << nodep);
+
+    // We can do this 2 ways. Either add a continuous assignment between
+    // the pin expression from the instance and the port variable, or
+    // simply inline the pin expression in place of the port variable.
+    // We will prefer to do the later whenever possible (and sometimes
+    // required). When inlining, we need to create an alias for the inined
+    // variable, in order to resovle hierarchical refrences against it
+    // later in V3Scope (and also for tracing, which is inserted later).
+    const bool inlineIt = [nodep]() {
```

Comment:
The outer function is multiple screen fulls, let's make it more reasonable by making this a normal function.